docs/sources/alerting/unified-alerting/_index.md

```diff
@@ -24,3 +24,29 @@ Before you begin using Grafana alerting, we recommend that you familiarize yours
 
 - The Grafana alerting system can retrieve rules from all available Prometheus, Loki, and Alertmanager data sources. It might not be able to fetch rules from other supported data sources.
 - We aim to support the latest two minor versions of both Prometheus and Alertmanager. We cannot guarantee that older versions will work. As an example, if the current Prometheus version is `2.31.1`, we support >= `2.29.0`.
+
+## Performance Considerations.
+
+Grafana Alerting now supports multi-dimensional alerting, or templatized alerting, so that one alert rule can generate many separate alerts. For example, one alert rule could generate an alert for each VM that's CPU is maxed out. This new capability adds some new performance considerations.
+
+Evaluating Alerting rules consumes RAM and CPU to compute the output of an alerting query, and network resources to write the results to the Grafana SQL database. The SQL database's network connections are often saturated before RAM or CPU are exhausted. The configuration of alert rules affects how many resources the Alertmanager consumes, and therefore the maximum number of rules a given configuration can support.
```

Comment:
```suggestion
Evaluating Alerting rules consumes RAM and CPU to compute the output of an alerting query, and network resources to write the results to the Grafana SQL database. The SQL database's network connections are often saturated before RAM or CPU are exhausted. The configuration of alert rules affects the maximum number of rules a given Grafana configuration can support.
```

Grafana can't provide a view per component, how do you feel about leaving the Alertmanager out of the picture?

Reply:
I think I used "Alertmanager" to cover too many things. I'm rephrasing to use "alerting" in most places.